### ISSUE TITLE
[Godot 4.x] Memory leak

### ISSUE THREAD

**Issue by git2013vb** [godotengine/godot#75613]
### Godot version

Any godot 4.x is affected. I didn't check 3.x

### System information

Debian 11

### Issue description

The Godot editor have memory leak. 
To check it run any project and wait for it (look at its process) 
It grow ~20Mb hour.

### Steps to reproduce

See before.

### Minimal reproduction project

NA

**Comment by YuriSizov** [comment]
> any project

Do you have an example?

**Comment by git2013vb** [comment]
> > any project
> 
> Do you have an example?

Well my projects are fine. The Godot editor leak. I noticed by chance because I have two projects (auth+gtw section) in another pc and I let them running for days - in production they will running forever -;) Their editors grow up to 5Gb each while it start with ~400Mb. Still my projects remain the same. I tested it with other projects and other versions - as stated before -. And in all cases the project is ok. while the editor process grow.
Well I use network ofc for all my projects but I tested them without any network connections. You cannot see any differences in the first 2~3 mins. You have to wait. My intuition tell me it happen when the editor refresh itself. But is just my intuition, so I can be wrong because I don't know the whole codebase.

**Comment by Taobis** [comment]
Yes, happens to me too. I am on Mac OS Ventura 13.2.1. I am leaving the Editor open all the Time and it caps the memory of my machine (8GB). When the Godoteditor starts lagging i know memory is full. I check in Activity Monitor and it is at around 6GB. I think my machine cannot allocate more. If i restart the editor my project starts at 1.1 GB.

**Comment by jitspoe** [comment]
While this issue was created first, there's more info over here, which is pretty much a duplicate: https://github.com/godotengine/godot/issues/96598
